### ISSUE TITLE
MultiSelect.Filterable a11y issue (Text contrast of 1.71 with its background is less than the WCAG AA minimum requirements...)

### ISSUE THREAD

**Issue by JensenAlexander** [carbon-design-system/carbon#7686]
Using chrome with the IBM Equal Access Accessibility Checker.

Setting the disabled property of the Multiselect.Filterable component gives:
- Text contrast of 1.71 with its background is less than the WCAG AA minimum requirements for text of size 12px and weight of 400

Reproduce by:

- Going to https://react.carbondesignsystem.com/?path=/story/multiselect--filterable
- Selecting Knobs
- Selecting Disabled

![image](https://user-images.githubusercontent.com/29494579/106191979-91ccfe80-6179-11eb-97fb-07c36793b7fa.png)

**Comment by tw15egan** [comment]
According to IBM Accessibility, disabled states do not have to meet the contrast minimum. You can read about it in this slack post on their channel https://ibm-studios.slack.com/archives/C036P1CTN/p1549289845286600?thread_ts=1549264906.285600&cid=C036P1CTN

The visual of disabled states is a fine line. It needs to look obviously disabled so the user doesn't mistakenly click on it and gets frustrated with why it is not working. Disabled states will also not be read by a screen reader so making them visually accessible will still not resolve it being accessible to all users. I hope this answers your question.

See also on the [WebAIM webpage](https://webaim.org/articles/contrast/#incidental) under `Exceptions -> Incidental -> Inactive`
